CPAC/nuisance/nuisance.py

```diff
@@ -264,14 +264,26 @@ def gather_nuisance(functional_file_path,
         regressor_file = censor_file_path
 
         if not regressor_file:
-            raise ValueError("Regressor type Censor specified in selectors but "
-                             "the corresponding file was not found!")
-
-        try:
-            censor_volumes = np.loadtxt(regressor_file)
-        except:
-            raise ValueError("Could not read regressor {0} from {1}."
-                             .format(regressor_type, regressor_file))
+            # ↓ This section is gross and temporary ↓
```

Comment:
sometimes I feel like "gross and temporary" keeps the digital world running 😄

Reply:
Yeah…

I'm not sure if we want to log that message and/or give a better message at some point. Right now, it just prints the message, generates the nothing-to-censor-array and keeps going.